.github/workflows/publish-release.yml

```diff
@@ -91,25 +91,3 @@ jobs:
       - id: get-release-version
         shell: bash
         run: ./scripts/get.sh ".version" "RELEASE_VERSION"
-
-  publish-release-to-gh-pages:
```

Comment:
We can remove the `get-release-version` above this as well. it's only used as an input to these two steps